### DISCUSSION TITLE
monorepo组合rc组件

### DISCUSSION THREAD

**Discussion by cheapCoder** [ant-design/ant-design#39699] [Ideas]
很好奇antd有讨论过用monorepo的形式组合rc-组件吗？
- 这样能借助工具处理依赖之间的版本升级(比如rc-motions依赖了[rc-menu](https://github.com/react-component/mentions/blob/master/package.json#L62))，
- 能抽离出util包给其他monorepo子包公用
- 子包能公用一些配置文件(eslint, prettier等)
- 没尝试过但相信也能做到子包单独热开发

**Comment by yoyo837** [comment]
可以是可以, 看必要性, 至少有一条: 懒得动. 😂

**Reply by cheapCoder** [reply]
这个pr除了官方应该没人提的动吧😂

**Reply by yoyo837** [reply]
个人认为有兴趣可以提,但最好问一下意见,免得白费功夫. /狗头
